amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/files/DeclareAndFileDocumentAsRecordTests.java

```diff
@@ -256,7 +256,7 @@ public void declareAndFileToValidLocationWithSpacesUsingActionsAPI() throws Exce
      * When I execute the action
      * Then I receive an error indicating that I have attempted to declare and file a document into an invalid record folder
      * And the document is not declared as a record
-     */
+
     @Test (dataProvider = "invalidDestinationPaths")
     public void declareAndFileToInvalidLocationUsingActionsAPI(String containerPath, String expectedException) throws Exception
     {
```

Comment:
Is this supposed to be commenting out this method as well?

Reply:
Yeap. This version does not have the ignore annotation yet, so I commented the test (declareAndFileToInvalidLocationUsingActionsAPI). Ignore was added by Tom here: https://github.com/Alfresco/alfresco-community-repo/pull/2224/commits/4456e4835fed275f2046bea771c464f43287edf7